ledger/trie/trie.go

```diff
@@ -104,6 +104,13 @@ func (t *Trie) Mutate(paths []ledger.Path, payloads []ledger.Payload) (*Trie, er
 	group.count = 0
 	group.node = &t.root
 
+	// We make a copy of the leftmost path in the group, and then keep a pointer
+	// to it in our group. This allows us to later garbage collect poth the slice
```

Comment:
This code is already outdated unfortunately :/ When reviewing, I'd suggest using the PR `Files changed` view and checking the files that have been reviewed as you go, so that you can use this view again later on to see only the changes since your last visit, while still having the latest version of the code.

Reply:
Yes, I moved from commit to commit, thanks!